resources/views/app/includes/contents/favorites.blade.php

```diff
@@ -138,7 +139,7 @@
         <!-- end card -->
 
         @empty
-            <p>No Episodes to display</p>
+            <p>Favourite episodes not available </p>
```

Comment:
*Code functionality - Critical*

@andela-araimi 

Write tests for this to assert that when a user clicks on favorite episodes, they get the 'Favorite episodes not available'.

Also, Consider rephrasing the message. `Not available` may refer to an error, not zero favorites.